src/openforms/emails/tests/test_tasks_integration.py

```diff
@@ -210,6 +214,64 @@ def test_email_sent_when_there_are_failures(
                 "https://api.brk.kadaster.nl/invalid/kadastraalonroerendezaken?postcode=1234AB&huisnummer=1",
                 status_code=400,
             )
+            m.get(
```

Comment:
why doesn't this use VCR? You can use `freeze_time` to pin our own timestamps, no?